src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncSpec.cs

```diff
@@ -336,13 +339,13 @@ public void A_Flow_with_SelectAsync_must_handle_cancel_properly()
         [LocalFact(SkipLocal = "Racy on Azure DevOps")]
         public void A_Flow_with_SelectAsync_must_not_run_more_futures_than_configured()
         {
-            this.AssertAllStagesStopped(() =>
+            this.AssertAllStagesStopped(async() =>
             {
                 const int parallelism = 8;
                 var counter = new AtomicCounter();
                 var queue = new BlockingQueue<(TaskCompletionSource<int>, long)>();
                 var cancellation = new CancellationTokenSource();
-                Task.Run(() =>
+                await Task.Run(() =>
```

Comment:
Is this supposed to be awaited? is the original intent to run it as a detached task?